lms/static/sass/shared/_footer.scss

```diff
@@ -202,6 +202,19 @@
         cursor: initial;
       }
     }
+
+    .select-lang-button {
```

Comment:
Question on where this custom CSS came from? Is it based on another button elsewhere?

Reply:
yes. It is the same that is used on https://www.edx.org/course for Schools & Partners dropdown button